docs/publishers/embedding.rst

```diff
@@ -44,3 +44,30 @@ document and have the same origin.
 
 The client will watch for new iframes being added to the document and will
 automatically enable annotation for them.
+
+
+Clipboard permissions when loaded in an iframe
+----------------------------------------------
+
+There is a few places in the sidebar where the user can copy content to the
+clipboard:
+
+* The link, when sharing an annotation.
+* The sidebar's version.
+* The content, when exporting annotations to any format.
+* Etc.
```

Comment:
I think this can be shortened, eg. "There are a few places in the sidebar where the user can copy content to the clipboard, such as exporting or copying links to annotations."